### PR TITLE
Keep our preview size if we knew it, prevents shapeshifting of attachments

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers

### CODE REVIEW

File: shared/actions/chat.js [hunk 1]
```diff
@@ -547,12 +548,26 @@ function * _incomingMessage (action: IncomingMessage): SagaGenerator<any, any> {
               })
             }
           }
+
+          let messages = []
+          if (message.type === 'Attachment') {
+            const existingAttachment = yield select(_messageSelector, conversationIDKey, message.messageID)
```
Comment (chromakode) [inline shared/actions/chat.js]: :sheep: Could you please elaborate on the case where we don't get a message with a preview size? Perhaps in a comment?

File: shared/actions/chat.js [hunk 2]
```diff
@@ -365,7 +365,7 @@ function * _postMessage (action: PostMessage): SagaGenerator<any, any> {
     // Time to decide: should we add a timestamp before our new message?
     const conversationState = yield select(_conversationStateSelector, conversationIDKey)
     let messages = []
-    if (conversationState && conversationState.messages !== null && conversationState.messages.size > 0) {
+    if (conversationState && conversationState.messages !== null && conversationState.messages.size > 1) {
```
Comment (MarcoPolo) [inline shared/actions/chat.js]: I was mistaken. I read this code as doing `.size - 2`. This is not needed

File: shared/actions/chat.js (hunk 1 above)
Comment (MarcoPolo) [inline shared/actions/chat.js]: You are right, thank you! This code is not needed at all.

I must have confused myself somehow :(

**Comment by keybase-ci-visdiff** [comment]
The commits bca2dee62c80...298e5a9a395b introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Attachment_Message-Basic_Not_loaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485225585305-bca2dee62c80-298e5a9a395b-linux/Attachment_Message-Basic_Not_loaded-bca2dee62c80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485225585305-bca2dee62c80-298e5a9a395b-linux/Attachment_Message-Basic_Not_loaded-298e5a9a395b.png)  
   Attachment_Message-Basic_Not_loaded comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits bca2dee62c80...571549079b99 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Attachment_Message-Basic_Not_loaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485225727890-bca2dee62c80-571549079b99-linux/Attachment_Message-Basic_Not_loaded-bca2dee62c80.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485225727890-bca2dee62c80-571549079b99-linux/Attachment_Message-Basic_Not_loaded-571549079b99.png)  
   Attachment_Message-Basic_Not_loaded comparison
</summary>

File: shared/actions/chat.js (hunk 1 above)
Comment (chromakode) [inline shared/actions/chat.js]: I'm glad I asked... I was feeling pretty befuddled!

**Comment by MarcoPolo** [comment]
@chromakode bump

**Pull request merged**